.github/contributing.md

```diff
@@ -50,67 +42,84 @@ To begin:
 
 3. Create a new branch on your fork to start working on your changes:
 
-   ```
-   git checkout -b MY_BRANCH_NAME
+   ```bash
+   git checkout -b <YOUR BRANCH NAME>
    ```
 
 4. Install the dependencies:
-   ```
+   ```bash
    pnpm install
    ```
    If you are on windows, use the `--ignore-scripts` flag
-   ```
+   ```bash
    pnpm install --ignore-scripts
    ```
 
-Now you have got the repo on your local machine, and you're ready to start making your changes!
+Now you have the repo on your local machine, and you're ready to start making your changes!
 
 <br/>
 
 ### Test Your Changes
 
-We use [yalc](https://github.com/wclr/yalc) to test changes locally.
+#### Writing Unit Tests
 
-Install the yalc CLI globally:
+Try to include unit test coverage with your changes where appropriate. We use `vitest` for testing, look for files with the `.test.ts` extension for examples of how to structure your unit tests.
+
+To run your tests, run the following from the package directory (most likely /packages/thirdweb):
 
 ```bash
-pnpm add yalc -g
+pnpm test <YOUR TEST FILE PATH>
```

Comment:
can also just simply switch it on `CI=true` -> meaning if `CI=true` we run with coverage otherwise not (we can set that up in the vitest.config.js settings)